dialoptions.go

```diff
@@ -210,20 +210,30 @@ func WithReadBufferSize(s int) DialOption {
 // WithInitialWindowSize returns a DialOption which sets the value for initial
 // window size on a stream. The lower bound for window size is 64K and any value
 // smaller than that will be ignored.
+//
+// Deprecated: use WithInitialStreamWindowSize to set a stream window size without disabling
```

Comment:
You can enclose symbol named in `[]`, (e.g. `[WithInitialStreamWindowSize]`) to link them to other symbols.

See https://go.dev/doc/comment#doclinks